File: desktop/src/ru/deathcry/spaceago/desktop/DesktopLauncher.java
package ru.deathcry.spaceago.desktop;

import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;
import ru.deathcry.spaceago.Main;

public class DesktopLauncher {
	public static void main (String[] arg) {
		LwjglApplicationConfiguration cfg = new LwjglApplicationConfiguration();
		cfg.title = "hello-world";
		cfg.width = 440;
		cfg.height = 800;
		new LwjglApplication(new Main(), cfg);
	}
}

File: desktop/src/org/nullpointerid/spaceago/desktop/DesktopLauncher.kt
package org.nullpointerid.spaceago.desktop

import com.badlogic.gdx.backends.lwjgl.LwjglApplication
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration
import org.nullpointerid.spaceago.SpaceShooter
import org.nullpointerid.spaceago.config.GameConfig

object DesktopLauncher {
    @JvmStatic
    fun main(arg: Array<String>) {
        val config = LwjglApplicationConfiguration()
        config.width = GameConfig.WIDTH
        config.height = GameConfig.HEIGHT
        config.title = GameConfig.TITLE
        config.resizable = false
        LwjglApplication(SpaceShooter, config)
    }
}
